src/qt/overviewpage.cpp

```diff
@@ -283,7 +283,7 @@ void OverviewPage::on_btnWebsite_pressed()
 
 void OverviewPage::on_btnBX_pressed()
 {
-    QDesktopServices::openUrl(QUrl("https://www.gridcoinstats.eu/block/"));
+    QDesktopServices::openUrl(QUrl("https://www.gridcoinstats.eu/block/#pk_campaign=GridcoinWallet&pk_kwd=" + QString::fromStdString(FormatFullVersion())));
```

Comment:
There is no need to have a slash before the #pk_campaign, but it doesn't matter either. 
You can just add the following after any URL towards the site
 `#pk_campaign=GridcoinWallet&pk_kwd=" + QString::fromStdString(FormatFullVersion())));`

So it can read anything like this
```
https://gridcoinstats.eu/block/#pk_campaign=GridcoinWallet&pk_kwd=1.2.3
https://gridcoinstats.eu/block/900000#pk_campaign=GridcoinWallet&pk_kwd=1.2.3
https://gridcoinstats.eu/tx/txHash#pk_campaign=GridcoinWallet&pk_kwd=1.2.3
```

The # sign just needs to be at the end of the address, doesn't matter what comes before it.